index.html

```diff
@@ -13,7 +13,7 @@
       <section class="navigator">
           <a href="#" class="mylogo">My logo</a>
           <nav>
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) for this project, " `Check that there is a title that adequately and briefly describes the content of the page.` ", please kindly update your page title to reflect the content of the page, currently it is `Skeleton project`, which is not descriptive of the page content. Checkout this [article](https://www.w3.org/WAI/test-evaluate/preliminary/#title) for more clarifications.